### PR TITLE
Allow user to specify which tables to use at command line.

### PR DESCRIPTION
**Pull request by lajones**
Depends on PR #3214. Further fixes for issue #2751. Updates the command-line and powershell command to allow the user to make use of the framework set up in PR #3214.

### CODE REVIEW

File: src/EntityFramework.Relational.Design/ReverseEngineering/TableSelectionSetBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using JetBrains.Annotations;
+
+namespace Microsoft.Data.Entity.Relational.Design.ReverseEngineering
+{
+    // TODO:
+    // Does not support schema or table names containing FilterSeparator.
```
Comment (natemcmaster) [inline src/EntityFramework.Relational.Design/ReverseEngineering/TableSelectionSetBuilder.cs]: Make an issue to track these?

**Comment by natemcmaster** [comment]
Tests?

File: src/EntityFramework.Relational.Design/ReverseEngineering/TableSelectionSetBuilder.cs (hunk 1 above)
Comment (lajones) [inline src/EntityFramework.Relational.Design/ReverseEngineering/TableSelectionSetBuilder.cs]: Will do.

File: src/EntityFramework.Commands/Commands/Program.cs [hunk 2]
```diff
@@ -143,6 +143,10 @@ public virtual int Main([NotNull] string[] args)
                             var provider = scaffold.Argument(
                                 "[provider]",
                                 "The provider to use. For example, EntityFramework.SqlServer");
+                            var tableFilters = scaffold.Option(
+                                "-t|--tables <filter>",
+                                "Selects for which tables to generate classes. "
+                                + "<filter> is a comma-separated list of schema:table entries where either schema or table can be * to indicate 'any'.");
```
Comment (bricelam) [inline src/EntityFramework.Commands/Commands/Program.cs]: We should discuss this syntax at a design meeting.

**Comment by lajones** [comment]
Closing this as the code compare had become confused. Opened #3230 instead.

**Pull request closed without merging**